src/call-context.ts

```diff
@@ -296,6 +296,8 @@ export class CallContext {
 
       const key = item.string();
 
+      this[ENV].free(Number(addr));
```

Comment:
Ooh, I think we'll want to update `free`'s signature to `number | bigint` and pass `addr` instead of `Number(addr)`. (`Number(addr)` might truncate the value.)

Reply:
thanks! I knew that `Number` cast felt wrong